src/main/scala/is/hail/annotations/aggregators/RegionValueAggregator.scala

```diff
@@ -4,6 +4,14 @@ import is.hail.annotations._
 import is.hail.expr.types._
 
 trait RegionValueAggregator extends Serializable {
+  var inputTyp: Type = _
+  var resultTyp: Type = _
+
+  def setTypes(input: Type, result: Type) {
```

Comment:
I'm really quite uncomfortable with this. It's much easier to get things wrong when the constructor of a class isn't sufficient to initialize it properly.

Reply:
I agree.